### ISSUE TITLE
UsersClient.View returning null, not throwing an exception

### ISSUE THREAD

**Issue by mzammitkg** [intercom/intercom-dotnet#140]
As of this morning, the following call to UsersClient seems to be returning null:

```

**Comment by wil55n** [comment]
Yes, we are experiencing the same issue as of today. Basically any userclient call is returning null.
We are on version 1.0.31 of the SDK.
If we update to 2.x of the SDK the call returns the correct data BUT we can not use 2.x right now because of the dependancy on .NET Standard

**Comment by abentabet** [comment]
The problem comes from the API and not the SDK, i've tried manual http request, it works by Id but not by userid, hope intercom fix that soon

**Comment by abentabet** [comment]
When using "ContactsClient.View" with .NET Framework 4.5.2 (same SDK version : 1.0.31) I have "null" but when using .NET Framework 4.7 it works ! ... we can't update .net framework because it impacts all our code

**Comment by mortenmoulder** [comment]
We're experiencing the exact same thing here. Works on 4.7.1 but not on 4.5.1. Using version 1.0.31.

I'm talking to Intercom as we speak. They are aware of the issue now, at least. Let's hope they fix this.

**Comment by RebliNk17** [comment]
Working with 4.7.1 and still receiving null...
